Source/Extensions/Date+DateOnly.swift

```diff
@@ -14,4 +14,12 @@ extension Date {
     let returnValue = calendar.date(from: newComponents)
     return returnValue!
   }
+    
+  /// Cuts off the time, leaving the beginning of the day for the new time zone
+  func dateOnly(calendar: Calendar, oldCalendar: Calendar) -> Date {
```

Comment:
I suggest renaming to something akin to  `func dateOnly(from: oldCalendar: Calendar, to: newCalendar: Calendar) -> Date {`